docs/Config_Reference.md

```diff
@@ -1704,6 +1704,96 @@ control_pin:
 #   See the "probe" section for information on these parameters.
 ```
 
+## [dockable_probe]
+
+Certain probes are magnetically coupled to the toolhead and stowed
+in a dock when not in use. One should define this section instead
+of a probe section if the probe uses magnets to attach and a dock
+for storage. See [Dockable Probe Guide](Dockable_Probe.md)
+for more detailed information regarding configuration and setup.
+
+```
+[dockable_probe]
+dock_position: 0,0,0
+#   The physical position of the probe dock relative to the origin of
+#   the bed. The coordinates are specified as a comma separated x, y, z
+#   list of values. Certain dock designs are independent of the z axis.
+#   if a z value is provided for these configurations, the z axis will
+#   be raised to that amount in order to prevent collisions.
+#   This parameter is required.
+approach_position: 0,0
+#   The X,Y position where the toolhead needs to be prior to moving into the
+#   dock so that the probe is aligned properly for docking or attaching.
+#   This parameter is required.
+detach_position: 0,0
+#   Similar to the approach_position, the detach_position is the X,Y
+#   where the toolhead is moved after the probe has been docked.
+#   For magnetically coupled probes, this is typically perpendicular to
+#   the approach_position in a direction that does not cause the tool to
+#   collide with the printer.
+#   This parameter is required
```

Comment:
it would be good to have an option to make them Z movement as well(for instance, make approach_position a little higher than probe dock, as additional safety)

Reply:
See previous comment